RELEASE_PROCESS.md

```diff
@@ -30,38 +30,86 @@ In order to release, you will need the following accounts/permissions:
+QUESTION: is this just updating ruby Gemfile.lock and erlang rebar.lock files? just delete the locks and regenerate? no updating of particular deps by hand?
+QUESTION: just make a new branch off of master to do this?
+- Make sure the omnibus build is into current channel. [explain?] <-----
   (This is triggered by expeditor once the build and tests in buildkite go through ok once a commit is merged to master)
-- Make sure the habitat builds are passing.
-- For updating just the patch verion refer the the section below on #Preparing for the release
+QUESTION: when/where does the merge to master happen? where is this step?
+One approach is to enter the following into a bash shell, where _version_ is the version number of the new release:
+```
+$ mixlib-install download chef-server -c current -a x86_64 -p ubuntu -l 16.04 -v <version>
+```
+Starting download https://packages.chef.io/files/current/chef-server/14.1.0/ubuntu/16.04/chef-server-core_14.1.0-1_amd64.deb
+- Make sure the habitat builds are passing. [pipeline? and when/where is this build kicked off?] <-----
+Chef / [chef/chef-server:master] habitat/build / master 
+Monitor the chef-server-notify slack channel for current progress ? <-----
+
+QUESTION: at some point @tas ran a build through the Chef / [chef/chef-server:master] omnibus/release pipeline.  where is this step represented?  explain.
+somewhere, insert (where?): The release pipeline runs automatically on merge after expeditor bumps the version...
 
 ### Testing the Release
 
-> This process is currently manual. There is working being done to fold this into a testing pipeline.
-
-Every merge to chef-server master is built and this clean build is tested
-with all pedant tests. We only run smoke tests for the FIPS mode. Upgrade
-and addon testing must be done in advance of the release.
-
-- [ ] Test as per the matrix from: https://docs.google.com/spreadsheets/d/1_gwxdrMnUiV8t2noi8zs1HyGivtIGMEx9KbYriIw7BY/edit#gid=536218507
-
-- [ ] If this release is being made to address a specific
-  high-urgency, high-severity customer issue or security issue, please
-  *specifically* test that the issue in question is fixed.
-
-If one of these tests has failed, you cannot ship a release until it's fixed.
-Note, no changes other than CHANGELOG/RELEASE_NOTES changes should
-land on master between testing and releasing since we typically tag
-HEAD of master. If something large does land on master, the release
-tag you create should point specifically at the build that you tested.
-The git SHA of the build you are testing can be found in
-`/opt/opscode/version-manifest.json`.
+Every merge to chef-server master must be built, and this build (master branch) must be tested with the Umbrella automated integration test pipeline: https://buildkite.com/chef/chef-umbrella-master-chef-server.
+The integration test run for the tag being shipped must be successful.
+
+QUESTION: i noticed the last release we ran a FULL Umbrella pipeline test:
```

Comment:
yes - it needs to be run before every release.

Reply:
For now we also nee dto manually log into manage to make sure it works as expected.